### ISSUE TITLE
FileNotFoundError: [WinError 2] The system cannot find the file specified

### ISSUE THREAD

**Issue by abdelrahman-orief** [jiaaro/pydub#306]
I am trying to open an mp3 file with the following command:
Here is the code

`from pydub import AudioSegment
finish = AudioSegment.from_mp3("ayat\\001001.mp3")`
It used to work then I added something to PATH yesterday and today it is not working at all.
I've tried everything: Change Path, add `AudioSegment.converter = r"C:\\ffmpeg\\bin\\ffmpeg.exe"` and it still doesn't work

- Python version: Python 3.6.5
- Pydub version: AudioSegment.converter = r"C:\\ffmpeg\\bin\\ffmpeg.exe"
- ffmpeg/avlib version: not sure

**Comment by Axel-Erfurt** [comment]
can you try the full path to 001001.mp3

"ayat\\001001.mp3" look wrong to me

**Comment by mohammadmomeni** [comment]
what happened ? I have same problem

**Comment by malet-ds** [comment]
I also have the same problem

**Comment by RussianDraco** [comment]
I think I solved this [here](https://stackoverflow.com/questions/68247327/cannot-find-a-solution-to-winerror-2-file-not-found-by-audiosegment-from-mp3)

**Comment by codehavoc** [comment]
@RussianDraco hey there, I'm facing the same issue and even opened another ticket about [this](https://github.com/jiaaro/pydub/issues/653). After few tries I tried the subprocessing module instead and met with the same root error that pydub provided. 

```
C:\Users\Win 10\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1435, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
```
I don't know what the source of the error anymore. Is this something happening because of an error with subprocessing module or ffmpeg? can some one tell me a way to fix this?

**Comment by qulx1** [comment]
@codehavoc Same as you if u got any solution give me it, please.

**Comment by codehavoc** [comment]
> @codehavoc Same as you if u got any solution give me it, please.

Hey there. If you did not manage to fix this issue yet, I manage to get my project running by temporarily setting ffmpeg to PATH using set command. But as you know you wont be able to close the cmd until you finished your work and every time you close the respective cmd you will have to re set the location to path before starting your work. 
If you have already found a more refined solution than this, Please let me know. thank you

**Comment by tt-thoma** [comment]
When importing the AudioSegment class, there is a warn that seems to be related to this issue:
```C:\Program Files\Python38\lib\site-packages\pydub\utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work```

**Comment by tt-thoma** [comment]
There are also quite a few issues from this error:
https://github.com/jiaaro/pydub/issues?q=Couldn%27t+find+ffmpeg+or+avconv
